broker/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu
+
+#
+# Installation as described in the server documentation
+# https://tunneldigger.readthedocs.io/en/latest/server.html
+#
+# Using the recommended apt practices.
+# https://docs.docker.com/develop/develop-images/dockerfile_best-practices/
+#
+RUN apt-get update && \
+    apt-get install -y \
+        iproute bridge-utils libnetfilter-conntrack-dev \
+        libnfnetlink-dev libffi-dev python-dev libevent-dev \
+        ebtables python-virtualenv wget && \
+    mkdir -p /srv/tunneldigger/tunneldigger/broker && \
+    cd /srv/tunneldigger && \
+    virtualenv env_tunneldigger
+#
+# Install OLSRD
+# https://github.com/OLSR/olsrd/
+#
+RUN apt-get -y install bison flex && \
+    mkdir -p /srv/olsrd && \
+    cd /srv/olsrd && \
+    wget -O- https://github.com/OLSR/olsrd/archive/v0.9.0.3.tar.gz \
+        | tar zxf - && \
+    cd * && \
+    make build_all && \
+    make install_all
```

Comment:
I don't remember doing this on my servers where I run tunneldigger. Why is this needed?

Reply:
https://tunneldigger.readthedocs.io/en/latest/server.html#routing-daemon says

> The example hook scripts require that the routing daemon (like olsrd) be configured with the Tunneldigger bridge interfaces.

I have no idea and so I added olsr.

Reply:
That's a very environment-specific part of the configuration. To my knowledge, most Freifunk Communities use Batman instead of OLSR, and would have to configure that.

I don't think it is possible to provide something in Docker that would make this reusable. But including OLSR here makes little sense IMO. People can derive their own Docker images that contains tunneldigger + whatever they need.

I can't speak for other communities, but in Freifunk Saar I don't think it would be possible to isolate tunneldigger from the rest of the network stuff that is going on on a gateway. All the routing table magic and bridge configuration and batman needs to happen in the same network namespace, so it cannot be easily split across containers.

Reply:
Oh. Thanks for clarifying this. 
I checked that point above. Other images can build on top of the provided tunnel digger image.